microscopium/features.py

```diff
@@ -95,10 +96,10 @@ def nearest_neighbors(lab_im, n=3, quantiles=[0.05, 0.25, 0.5, 0.75, 0.95]):
     """
     if lab_im.dtype == bool:
         lab_im = nd.label(lab_im)[0]
-    centroids = np.array([p.centroid
-                          for p in measure.regionprops(lab_im, coordinates='rc')])
-    nbrs = (NearestNeighbors(n_neighbors=(n + 1), algorithm='kd_tree').
-                         fit(centroids))
+    centroids = np.array(
```

Comment:
This change is purely because 'auto-format on save' kept altering these lines. It's not of substance.

Reply:
"auto-format on save" — can you change the settings on your IDE as to what that does? We are moving to using `black -S -l 79` in napari.